<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtDialog</class>
 <widget class="QDialog" name="QtDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CppQwtExample5</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QwtPlot" name="plot" native="true"/>
   </item>
   <item row="2" column="1">
    <widget class="QSlider" name="alpha">
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSlider" name="beta">
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_alpha">
     <property name="text">
      <string>Alpha</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_beta">
     <property name="text">
      <string>Beta</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_dt">
     <property name="text">
      <string>dt</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSlider" name="dt">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="value_alpha">
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="value_beta">
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="value_dt">
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="box_type">
     <item>
      <property name="text">
       <string>Alpha filter</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Alpha  Beta filter</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Filter type</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QWidget</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>box_type</tabstop>
  <tabstop>alpha</tabstop>
  <tabstop>beta</tabstop>
  <tabstop>dt</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
